modules/aws-load-balancer-operator-install.adoc

```diff
@@ -0,0 +1,340 @@
+// Module included in the following assemblies:
+//
+// * networking/aws-load-balancer.adoc
+
+:_content-type: PROCEDURE
+[id="aws-load-balancer-operator-install_{context}"]
+= Installing an AWS Load Balancer (ALB) Operator
+Users that wish to install the AWS Load Balancer (ALB) Operator can do so as long as the following prerequisites are met.
+
+.Prerequisites
+
+* A multi-AZ ROSA cluster deployed with STS and created using customer-managed Subnets
+* Access to modify the VPC and Subnets of the created {product-title} cluster
+// * AWS account credentials
+* Configured {product-title} (ROSA) CLI
+* Configured Amazon Web Services (AWS) CLI
+* Configured OpenShift (oc) CLI
+* OpenShift Container Platform (OCP) 4.13 or later
+
+
+[NOTE]
+====
+When installing for use in an AWS Local Zone, the Local Zone is enabled for the account and the ALB Operator is available within the Local Zone/region.
```

Comment:
It is not "the ALB Operator is available within the ...".
It is "the Application Load Balancer (ALB) is an available feature within the Local Zone"

Reply:
> It is not "the ALB Operator is available within the ...". It is "the Application Load Balancer (ALB) is an available feature within the Local Zone"

Ack. Changed.